backend/routes/userRoutes.js

```diff
@@ -17,6 +17,10 @@ router.post('/', async (req, res) => {
     return res.status(400).json({ error: 'Username already exists' });
   }
 
+  if (username.includes(null)){
+    return res.status(400).json({ error: 'Username can not contain "null"' });
+  }
```

Comment:
What if I wanna create a username called, "neverbenull"? Would the system throw 400?